tests/torch_datasets/test_site.py

```diff
@@ -118,22 +101,62 @@ def test_convert_from_dataset_to_dict_datasets(sites_dataset):
         assert key in sample
 
 
-def test_site_dataset_with_dataloader(sites_dataset) -> None:
-    expected_coods = {
-        "site__date_cos",
-        "site__time_cos",
-        "site__time_sin",
-        "site__date_sin",
-    }
+def site_collate_fn(batch_xr_datasets):
```

Comment:
Is this wjat we do in save samples? I think we dont do this. 
Might be good to test what we do do in save_samples